### PR TITLE
html css completion

### PR DESCRIPTION
**Pull request by salvik21**

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -114,4 +114,95 @@ main {
...
+  width: 44.25%;
+  margin: 10px 10px;
+}
+
+.gallery__img-wrapper {
+  margin: 0 10px;
+}
+
+.gallery {
+  background-color: rgb(250, 250, 250);
```
Comment (YuraLeo94) [inline style.css]: For what reason are you using here, above and below `rgb`?

File: style.css [hunk 2]
```diff
@@ -114,4 +114,95 @@ main {
...
+  margin-bottom: 10px;
+}
+
+.galery__title, .gallery__subtitle, .footer__title, .footer__subtitle, .footer__copyright {
+  text-align: center;
+}
+
+.gallery__img-wrapper img {
+  width: 44.25%;
+  margin: 10px 10px;
```
Comment (YuraLeo94) [inline style.css]: can do shorter ` margin: 10px`

File: style.css [hunk 3]
```diff
@@ -114,4 +114,95 @@ main {
...
+}
+
+.about__text-wrapper {
+  padding: 20px;
+}
+
+.about__subtitle, .gallery__subtitle {
+  text-transform: uppercase;
+  color: #be4642;
+  margin: 10px 0;
```
Comment (YuraLeo94) [inline style.css]: change please on next size `margin: 15px 0 30px;`

File: style.css [hunk 4]
```diff
@@ -114,4 +114,95 @@ main {
...
+.about__paragraph {
+  color: rgb(58, 57, 57);
+  font-family: Montserrat, sans-serif;
+}
+
+.about__title, .galery__title {
+  font-family: monospace;
+  font-weight: bold;
+  font-size: 38px;
+  margin-bottom: 10px;
```
Comment (YuraLeo94) [inline style.css]: change please on next size `margin: 0`, don't need `margin` for this element

File: style.css [hunk 5]
```diff
@@ -114,4 +114,95 @@ main {
...
+  margin: 10px 10px;
+}
+
+.gallery__img-wrapper {
+  margin: 0 10px;
+}
+
+.gallery {
+  background-color: rgb(250, 250, 250);
+  padding: 30px 0;
```
Comment (YuraLeo94) [inline style.css]: change please on next size `padding: 70px 0;`

File: style.css [hunk 6]
```diff
@@ -114,4 +114,95 @@ main {
...
+  margin-top: 30px;
```
Comment (YuraLeo94) [inline style.css]: here don't need `margin-top`, since padding give you necessary space

**Review by YuraLeo94 — CHANGES_REQUESTED** [review]
The gallery does not match the [design ](https://github.com/Hetachi/FrontEndTraining/blob/master/visuals/Renome%20Template/!!!Mobile/One%20Page/Mobile_one%20page.psd)- fix it please
image
Need space between icons looks [design ](https://github.com/Hetachi/FrontEndTraining/blob/master/visuals/Renome%20Template/!!!Mobile/One%20Page/Mobile_one%20page.psd)please
image
Why do you use new font for the logo in the footer?  Looks please difference between the logo in the header and footer in the design. Your old logo font from the header will be fine for the footer logo.
image

**Review by salvik21 — COMMENTED** [review]
Снимок

**Review by salvik21 — COMMENTED** [review]
Снимок2

File: style.css [hunk 7]
```diff
@@ -114,4 +114,107 @@ main {
...
+.gallery__img img {
+  width: 100%; 
+  display: block;
+}
+
+.gallery__img-wrapper {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  margin: 30px 5px 50px 5px;
```
Comment (YuraLeo94) [inline style.css]: Change please at the `margin: 0 5px` it will be more close to the design.

**Review by YuraLeo94 — CHANGES_REQUESTED** [review]

**Review by YuraLeo94 — APPROVED** [review]

**Pull request merged**